### DISCUSSION TITLE
HMS-600-2T  // Nutzung der Eingänge

### DISCUSSION THREAD

**Discussion by rs3hc** [hoylabs/OpenDTU-OnBattery#693] [General]
Hallo Leute.

Ich baue gerade ein BKW mit OpenDTU-OnBattery.

Nun habe ich einen HMS-600-2T, der ja bekanntlich zwei Eingänge mit zwei Trackern hat die auch seperat in Opendtu erkannt werden.

An Eingang 1 würde ich gern das 48v Batteriesystem hängen.

Jetzt war mein Gedanke an Eingang 2 noch 1-2 PV Module direkt anzustecken.

Wie verhält es sich jetzt, wenn z.b. 500W AC benötigt würden und 400W an Eingang 2 erzeugt würden?

Werden die Eingänge 50/50 belastet oder lässt sich das Regeln?

LG aus dem Erzgebirge

**Comment by spcqike** [comment]
Nach aktuellem Stand regeln die hoymiles immer die MPPT gleicher Maßen. 

Wenn du bspw 500W AC möchtest würde auf jedem tracker 250W AC bereitgestellt. 

Davon abgesehen kann jeder einzelner tracker nur 1/n der Gesamtleistung. In deinem Fall 300W. 

Auch wenn die Module in Parallelschaltung >300W liefern, bekommst du so nur 300W heraus. 

Ein Mischbetrieb von Batterie und PV an einem wechselrichter ist aktuell nicht vorgesehen. Die wechselrichter können es einfach nicht differenzieren. Man müsste die Batterie, wenn man sie nicht leeren möchte, hart trennen und später wieder zuschalten (einschaltstrom Thematik) 
dazu werden eventuelle PV Überschüsse verschenkt, weil der Eingang nicht mehr als 1/n der maximalen Leistung verarbeiten kann.

**Reply by rs3hc** [reply]
Ah ok, dh heißt ich müsste sowieso beide Eingänge parallel auf die Batterie schalten.

Und für die anderen Platten einen separaten WR nutzen.

Da wäre es in der Tat einfacher die anderen Platten über einen weiteren Victron mit auf die Batterie zu geben.

**Reply by spcqike** [reply]
> Da wäre es in der Tat einfacher die anderen Platten über einen weiteren Victron mit auf die Batterie zu geben.

das wäre auch meine favorisierte Option. So kannst du die gesamte PV Energie zum Laden nutzen und die Regelung zur Nulleinspeisung ist einfacher, wenn du nur einen wechselrichter regeln musst. 

Mit zwei wechselrichtern wäre es aktuell so, dass einer fest einspeist und der andere geregelt wird. Damit kannst du aber eben auch Überschüsse produzieren und einspeisen (wenn der fest laufende deinen Grundbedarf übererfüllt, würde er ins Netz einspeisen, der geregelte würde gestoppt)

**Reply by rs3hc** [reply]
> das wäre auch meine favorisierte Option. So kannst du die gesamte PV Energie zum Laden nutzen und die Regelung zur Nulleinspeisung ist einfacher, wenn du nur einen wechselrichter regeln musst. 
> 

Ja, da wäre aber immernoch das "Nice-to-have" mit den zwei MPPT Auswertungen 😅

**Comment by bikerbiker12** [comment]
Ich habe meine 4 Module zu je 2 x seriell und dann parallel an einen Victron Laderegler, dieser lädt dann mein 48V Akku. Der Hausbedarf wird eingespeist und der Rest geht in den Akku. Wenn der Akku voll ist, geht der Überschuss direkt ins Haus, was dann noch übrig bleibt ist entweder Verlust oder du stellst es so ein, dass ins Netz eingespeist wird, oder schaffst mehr Speicherkapazität. Auf der Victron Seite hast du einen MPPT Rechner, mit dem kannst du dir berechnen welchen du benötigst (du benötigst alle Werte der Module). 
Dann nimmst du dir auf jeden Fall einen größeren damit du vielleicht noch Module dran hängen kannst!
